### DISCUSSION TITLE
How to Download and Install mockoon version 1.17.0 on mac?

### DISCUSSION THREAD

**Discussion by LahLoh** [mockoon/mockoon#810] [Help]
How to Download and Install mockoon version 1.17.0 on mac?

**Comment by 255kb** [comment]
You can find the binaries on the [repository releases](https://github.com/mockoon/mockoon/releases). But we only support the last 4 versions. The binaries shouldn't be available for V1.17.0

**Reply by LahLoh** [reply]
Hi Guillaume, thanks for the response.

Is there any way to be able to use mockoon version 1.17.0 on mac?

**Reply by 255kb** [reply]
You would have to rebuild it from the source. 
Any precise reason you need an older version?

**Reply by LahLoh** [reply]
I tried to build from source code, but failed. (checkout from tag 1.17.0)
i'm using node v16.17.0 and npm 8.15.0,

<img width="944" alt="Screen Shot 2022-09-01 at 19 21 13" src="https://user-images.githubusercontent.com/110376807/187912558-8891105d-9f37-4794-b40e-6f0b734d5344.png">

i'm try using different node version (12.x.x and 14.x.x), but still error
<img width="1217" alt="Screen Shot 2022-09-01 at 20 25 14" src="https://user-images.githubusercontent.com/110376807/187925420-01e15bfb-2335-4755-b0b4-0bd4d9a3b316.png">


the reason is in my company using mockoon version 1.17.0.

**Reply by 255kb** [reply]
I don't see where the first error is coming from, if you ran npm install, electron-builder should be here.
Following the contributing guidelines should be enough to build the Mac binaries. You will have to disable the signing and notarization in electron-builder config to make it work.
Sorry, I cannot help building an older version as I am currently not in the office for several days. And I can hardly do this remotely.
